blog/2018/12/21/a-little-christmas-template-cooking/index.markdown

```diff
@@ -0,0 +1,104 @@
+---
+status: published
+title: Day 21: A Little Christmas Template Cookings
+author: brian d foy
+tags:
+    - templates
+    - advent
+images:
+  banner:
+    src: '/blog/2018/12/21/a-little-christmas-template-cooking/banner.jpg'
+    alt: 'Elgin Theater'
+    data:
+      attribution: |-
+        <a data-flickr-embed="true"  href="https://www.flickr.com/photos/39908901@N06/11412118604/in/photolist-ios6gA-5LGoKq-PnBoBQ-ibEPHM-7qUncc-SSGFhT-SCYSD9-7pwexw-2doKrZi-b18y32-acMecE-91MNrB-ogyG6P-LX7iMB-ShNXEj-8ZydMc-qdCE4G-4gn2We-qeqCfM-vcGZh-uVByK-h7ffe3-5y5KW9-2cvWMiq-G28Wj6-9ghgVP-4kkcHa-95mLjQ-45ntcs-ShQc5J-nKVW5Q-SP6zgo-imwWqF-SFiTgF-b1xMUr-98xU34-iy3ziA-3ajXVu-7qujMF-SFiitR-dD7wZD-nZfia-kyWT4M-iAZSYf-8ys5cz-94snpg-ShNKW9-5EV5Ns-21qzGXJ-b3oaiT" title="Peanut Butter Kisses cookies"><img src="https://farm8.staticflickr.com/7346/11412118604_cd0ee37d7c_k.jpg" width="2048" height="1356" alt="Peanut Butter Kisses cookies"></a>
+data:
+  bio: briandfoy
+  description:
```

Comment:
description is missing. The description is used for the twitter card text (along with the title).